samples/CognitiveSearch/readme.md

```diff
@@ -12,6 +12,18 @@ input-file:
   - https://github.com/Azure/azure-rest-api-specs/blob/e606243e5297312781dd7dbfd7ab76d2329cc088/specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json
 namespace: Azure.CognitiveSearch
 
+suppress-abstract-base-class:
+- Analyzer
+- CharFilter
+- CognitiveServicesAccount
+- DataChangeDetectionPolicy
+- DataDeletionDetectionPolicy
+- ScoringFunction
+- Similarity
+- Skill
+- TokenFilter
+- Tokenizer
```

Comment:
How does this configuration work with the rename configurations?
We should fill in its generated name or its original name in the swagger?
Could we emphasize this part in its corresponding documentation?
Since this is not a mgmt-only configuration, do we have a place for its documentation?

Reply:
The original name, unfortunately...

Reply:
@ArcturusZhang I've updated the `polish.md` to emphasize the exact name. See the latest commit.